packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -23,6 +23,10 @@ export const publicPrices = prices => {
   return { amountIn: prices.swapperGives, amountOut: prices.swapperGets };
 };
 
+/**
+ * @typedef {Record<string, Amount>} NotificationState
```

Comment:
Is the type really a record with arbitrary string keys? not `{ Central: Amount, Secondary: Amount, Liquidity: Amount}`?

Reply:
good call